Wrapping/macro_files/itk_end_wrap_module.cmake

```diff
@@ -497,20 +497,20 @@ ${DO_NOT_WAIT_FOR_THREADS_CALLS}
     # run doxygen
 
     # create the target doc dir
-    set(ITK_WRAP_DOC_LIBRARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/Doc"
+    set(ITK_DOXYGEN_OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/Doc"
```

Comment:
I think we should retain `ITK_WRAP_DOC_LIBRARY_DIR`, which is also used in other ways.

Instead of this change and the change at 513, can we just add `set(ITK_DOXYGEN_OUTPUT_DIR ${ITK_WRAP_DOC_LIBRARY_DIR})` before line 507?

Reply:
In the mentioned if block I saw the `ITK_WRAP_DOC_LIBRARY_DIR ` was set and at the end of the block it was unset again and didn't see any other places where it was used hence I chose to to replace the variable, it looks like that I oversaw 2 places (I was probably lucky that the directories already existed).
Fixed it.